src/components/modals/Modal.vue

```diff
@@ -17,9 +23,17 @@
 
 <script>
 export default {
+  data: function() {
+    return {
+      isMaximized: false
+    };
+  },
   methods: {
     close() {
       this.$emit("close");
+    },
+    toggleMaximized() {
+      this.isMaximized = !this.isMaximized;
```

Comment:
This is only called in one place in the template, so I'd suggest you inline it into the template.

Reply:
will change